### PR TITLE
Sohaib -Pydantic models- Authors , Language , Copies  

### PR DESCRIPTION
**Pull request by SohaibkhanFolio3**
Pydantic models for Authors language and copies , field description for each field added.
Ticket link:
https://app.asana.com/0/1204411152293714/1204439508142890/f
https://app.asana.com/0/1204411152293714/1204439508142888/f
https://app.asana.com/0/1204411152293714/1204439508142894/f

### CODE REVIEW

**Review by mustansirMuz — COMMENTED** [review]
Looks good, just these few changes:

- The docstrings should be inside the class
- Model names should have "Schema" prefix, so it is consistent across all our schemas
- Add example schema
- Add all fields in UserSchema

**Comment by SohaibkhanFolio3** [comment]
> Looks good, just these few changes:
> 
> * The docstrings should be inside the class
> * Model names should have "Schema" prefix, so it is consistent across all our schemas
> * Add example schema
> * Add all fields in UserSchema

Already did all this , and user schema is not done by me !!

**Comment by mustansirMuz** [comment]
> > Looks good, just these few changes:
> > 
> > * The docstrings should be inside the class
> > * Model names should have "Schema" prefix, so it is consistent across all our schemas
> > * Add example schema
> > * Add all fields in UserSchema
> 
> Already did all this , and user schema is not done by me !!

my bad, got confused because you had named the Author class as User. I see you have fixed that now.

**Review by mustansirMuz — APPROVED** [review]

File: src/schemas/author_schema.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    birth_date: datetime = Field(title="Date of Birth For The Author")
+    death_date: datetime = Field(title="Date of Passing For The Author", default=None)
+
+    @validator("death_date")
+    def birth_date_greater_than_death_date(cls, v, values, **kwargs):
+        if "birth_date" in values and values["death_date"] >= values["birth_date"]:
+            raise ValueError("Death Date must be greater than Birth Date")
+
+
+class Config:
```
Comment (mtalhatahir2001) [inline src/schemas/author_schema.py:26]: These config classes are subclasses of schema classes. So they should be inside schema models not out side.

**Review by mtalhatahir2001 — CHANGES_REQUESTED** [review]

**Review by mtalhatahir2001 — APPROVED** [review]
Approved.

**Review by Sana-Ullah786 — APPROVED** [review]
Approved. ✨

**Pull request merged**